packages/logo/src/Logo.story.tsx

```diff
@@ -103,5 +103,11 @@ storiesOf('Logo', module)
       );
     };
 
-    return <>{Object.keys(map).map(renderProductLogo)}</>;
+    return (
+      <>
+        {Object.keys(map).map(key =>
+          renderProductLogo(key as keyof typeof map),
```

Comment:
This is an interesting solution to this TS issue! We've definitely worked around this elsewhere in less elegant ways.